Tests/SRGDataProviderNetworkTests/ImageServicesTestCase.m

```diff
@@ -32,175 +32,110 @@ - (void)tearDown
 
 #pragma mark Tests
 
-- (void)testImageScalingDefault
+- (void)testFixedSizeImage
 {
-    XCTestExpectation *expectation = [self expectationWithDescription:@"Request succeeded"];
+    XCTestExpectation *expectation1 = [self expectationWithDescription:@"Request succeeded"];
     
     [[self.dataProvider mediaWithURN:kVideoURN completionBlock:^(SRGMedia * _Nullable media, NSHTTPURLResponse * _Nullable HTTPResponse, NSError * _Nullable error) {
         XCTAssertNotNil(media);
         XCTAssertNil(error);
         
-        NSURL *imageURL = [self.dataProvider URLForImage:media.image withWidth:SRGImageWidth320 scaling:SRGImageScalingDefault];
+        NSURL *imageURL = [self.dataProvider URLForImage:media.image withWidth:SRGImageWidth320 scalingService:SRGImageScalingServiceDefault];
         UIImage *image = [UIImage imageWithData:[NSData dataWithContentsOfURL:imageURL]];
         XCTAssertEqual(image.size.width, 320.);
         XCTAssertEqual(image.size.height, 180.);
         
-        [expectation fulfill];
+        [expectation1 fulfill];
     }] resume];
     
     [self waitForExpectationsWithTimeout:30. handler:nil];
-}
```

Comment:
If I well understand, here we are trying to test two things: 

1. Fixed size image for the `default` scaling service.
2. Fixed size image for the `centralized` scaling service.

What do you think if we split this test into two different tests?

### Proposition:

- `testFixedSizeImageWithDefaultScalingService`
- `testFixedSizeImageWithCentralizedScalingService`

Reply:
I agree, it seems more understandable.

Reply:
I propose to only focus on centralized one here. and take care of various image size options.
BU tests moved in the BU test files.